corehq/toggles.py

```diff
@@ -1724,9 +1724,12 @@ def _commtrackify(domain_name, toggle_is_enabled):
 )
 
 
-USER_API_USE_ES_BACKEND = StaticToggle(
-    'user_api_use_es_backend',
-    'Use new ES backend for User API.',
+# todo: remove after Dec 25, 2019 if no one is using
```

Comment:
Time to remove this?

(I was reading this file today, adding a new flag and feeling a little sad to be doing so, and noticed the comment.)

Reply:
Thanks for the reminder! https://github.com/dimagi/commcare-hq/pull/26506